jekyll/_cci2/release/configure-release-markers.adoc

```diff
@@ -0,0 +1,46 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Configure release markers
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to configure release markers in your workflow
+:icons: font
+:experimental:
+
+This tutorial shows how to add a "release marker" step to your workflow config. Release markers allow you log all new releases in one place and link back to the CI/CD pipelines that caused them, without needing to install the release agent.
```

Comment:
I think we could maybe make it clear here that this guide is for people who, for a specifc project at least, aren't using the wider releases features, to make it really clear it's separate from the rest of the setup guides. Also people won't have any knowledge of what the release agent is at this point. I can have a go if you like?

Reply:
Also I wonder which other docs would be a good place to link to here from to signpost. I guess the deployment docs

Reply:
Go ahead and give it a go! I think you're correct that deployment docs could make sense to link here, but I'm not sure about anything else.

Reply:
I like your updates. Thank you!